src/Webhooks/Core.cs

```diff
@@ -3,9 +3,11 @@
 using BrokeProtocol.API;
 using BrokeProtocol.Entities;
 using BrokeProtocol.Managers;
+using Newtonsoft.Json;
 using System;
 using System.Collections.Generic;
 using System.Diagnostics;
+using System.IO;//remove
```

Comment:
// remove?

Reply:
Added when testing so no need to have it